packages/fxa-graphql-api/src/backend/legal.service.ts

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ConfigService } from '@nestjs/config';
+
+import { AppConfig } from '../config';
+import { determineLocale } from 'fxa-shared/l10n/determineLocale';
+import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+import { Injectable } from '@nestjs/common';
+
+import request from 'request';
+
+const LEGAL_DOCS_PATH = '/settings/legal-docs';
+
+@Injectable()
+export class LegalService {
+  private settingsUrl: string;
+
+  constructor(
+    private log: MozLoggerService,
+    configService: ConfigService<AppConfig>
+  ) {
+    this.settingsUrl = configService.get(
+      'settingsUrl'
+    ) as AppConfig['settingsUrl'];
+  }
+
+  public async getDoc(locale: string, file: string) {
+    // Determine the best local to use, given the available ones
+    const availableLocales = await this.getAvailableLocales(file);
+    locale = determineLocale(locale, availableLocales);
+
+    let markdown = await this.tryGetDoc(locale, file);
+    if (!markdown && locale !== locale.replace('-.*', '')) {
```

Comment:
We might could use a comment here about this being tied to the external JSON provided by the legal-docs repo. If something ever breaks there in the future it might help us figure it out a hair faster.

Also kind of wondering if we should file an issue to create a constant in fxa-shared for `DEFAULT_LOCALE`, we do have a lot of scattered `'en'` references now...